yarn-project/types/src/interfaces/hasher.ts

```diff
@@ -3,27 +3,27 @@
  */
 export interface Hasher {
   /**
-   * Compresses two 32-byte hashes.
-   * @param lhs - The first hash.
```

Comment:
Although what we are hashing is usually the output of another hash, calling it hash here is quite confusing as the method name is also hash.